guides/security/device_admin_profiles.md

```diff
@@ -0,0 +1,31 @@
+# Admin profiles and standard user profiles on devices
+
+As part of our commitment to secure working practices, we install separate admin and standard user profiles on laptops. The reason this restriction is in place is to prevent the accidental downlaoding and installation of malicious software on company devices, to close an attack vector, and to protect both Made Tech and our customers.
```

Comment:
Minor typo.

```suggestion
As part of our commitment to secure working practices, we install separate admin and standard user profiles on laptops. The reason this restriction is in place is to prevent the accidental downloading and installation of malicious software on company devices, to close an attack vector, and to protect both Made Tech and our customers.
```